### PR TITLE
Fixing loggers and related package values

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Purpose
Fixing loggers and related package values

### CODE REVIEW

File: apim-apk-agent/internal/utils/apis_fetcher.go [hunk 1]
```diff
@@ -75,27 +75,27 @@ func FetchAPIsOnEvent(conf *config.Config, apiUUID *string, k8sClient client.Cli
 			// Read the .zip files within the root apis.zip and add apis to apiFiles array.
 			for _, file := range zipReader.File {
 				apiFiles[file.Name] = file
-				logger.LoggerSync.Infof("API file found: " + file.Name)
+				logger.LoggerUtils.Infof("API file found: " + file.Name)
```
Comment (Krishanx92) [inline apim-apk-agent/internal/utils/apis_fetcher.go:78]: Shall we make debug?

File: apim-apk-agent/internal/utils/apis_fetcher.go [hunk 2]
```diff
@@ -64,7 +64,7 @@ func FetchAPIsOnEvent(conf *config.Config, apiUUID *string, k8sClient client.Cli
 		GetAPI(c, nil, envs, sync.RuntimeArtifactEndpoint, true)
 	}
 	data := <-c
-	logger.LoggerMsg.Info("Receiving data for an API")
+	logger.LoggerUtils.Info("Receiving data for an API")
```
Comment (Krishanx92) [inline apim-apk-agent/internal/utils/apis_fetcher.go:67]: Lets remove this or add a debug log with api ID

File: apim-apk-agent/internal/messaging/notification_listener.go [hunk 3]
```diff
@@ -449,7 +449,7 @@ func handlePolicyEvents(data []byte, eventType string) {
 			RateLimitTimeUnit: subscriptionPolicyEvent.RateLimitTimeUnit, StopOnQuotaReach: subscriptionPolicyEvent.StopOnQuotaReach,
 			TenantDomain: subscriptionPolicyEvent.TenantDomain, TimeStamp: subscriptionPolicyEvent.TimeStamp}
 
-		logger.LoggerMsg.Infof("SubscriptionPolicy event data %v", subscriptionPolicy)
+		logger.LoggerMessaging.Infof("SubscriptionPolicy event data %v", subscriptionPolicy)
```
Comment (Krishanx92) [inline apim-apk-agent/internal/messaging/notification_listener.go:452]: here as well

File: apim-apk-agent/internal/eventhub/dataloader.go [hunk 4]
```diff
@@ -182,21 +181,21 @@ func InvokeService(endpoint string, responseType interface{}, queryParamMap map[
 		} else {
 			c <- response{err, nil, 0, endpoint, gatewayLabel, responseType}
 		}
-		loggers.Info("Error occurred while calling the REST API: "+serviceURL, err)
+		logger.LoggerEventhub.Infof("Error occurred while calling the REST API: "+serviceURL, err)
```
Comment (Krishanx92) [inline apim-apk-agent/internal/eventhub/dataloader.go:184]: Error

File: apim-apk-agent/internal/agent/agent.go [hunk 5]
```diff
@@ -222,17 +221,17 @@ OUTER:
...
 			switch s {
 			case os.Interrupt:
-				logger.LoggerInternalMsg.Info("Shutting down...")
+				logger.LoggerAgent.Info("Shutting down...")
 				break OUTER
 			}
 		}
 	}
-	logger.LoggerInternalMsg.Info("Bye!")
+	logger.LoggerAgent.Info("Bye!")
```
Comment (Krishanx92) [inline apim-apk-agent/internal/agent/agent.go:236]: Do we need this?

File: apim-apk-agent/internal/utils/apis_fetcher.go (hunk 2 above)
Comment (CrowleyRajapakse) [reply apim-apk-agent/internal/utils/apis_fetcher.go]: fixed

**Review by Krishanx92 — APPROVED** [review]

**Pull request merged**